.storybook/theme.js

```diff
@@ -0,0 +1,9 @@
+import { create } from "@storybook/theming";
+import { darkTheme } from "../src/themes";
+
+export default create({
+  base: "dark",
+  brandTitle: "OpenDEX storybook",
+  brandImage:
+    "https://gblobscdn.gitbook.com/spaces%2F-Lq7dAvRcXVJXcPrZrNF%2Favatar.png?alt=media",
```

Comment:
This image might stop working as new version is pushed and CDN cache is invalidated. Might need to add a more permanent link to OpenDEX logo image.